acceptance/tests/harvest_latest_versions.py

```diff
@@ -0,0 +1,175 @@
+    return harvesting_date, archives_ids
+
+
+def save_content_to_file(content, data_path, filename):
+    file_path = path.join(data_path, filename)
+    with open(file_path, "w") as f:
+        json.dump(content, f)
+
+
+def save_archives_ids_file(harvesting_date, archives_ids, data_path, filename):
+    archives_ids_json = {HARVESTING_DATE: harvesting_date, ARCHIVES_IDS: archives_ids}
+    save_content_to_file(archives_ids_json, data_path, filename)
+
+
+def get_entity_data(entity_id):
+    query = {ACTION: WBGETENTITIES, IDS: entity_id, FORMAT: JSON}
+    response = requests.get(API_URL, params=query)
+    response_json = response.json()
+    return response_json[ENTITIES][entity_id]
+
+
+def has_been_modified_since(catalog_data, last_harvesting_date):
+    modification_date = catalog_data[MODIFIED]
+    last_harvesting_date = dp.parse(last_harvesting_date)
+    modification_date = dp.parse(modification_date)
+    return modification_date > last_harvesting_date
+
+
+def harvest_archives_ids(catalog_data):
+    harvesting_date = catalog_data[MODIFIED]
+    archives_ids = []
+
+    sources = catalog_data[CLAIMS][SOURCE_PROPERTY]
+    for source in sources:
+        source_id = source[MAINSNAK][DATAVALUE][VALUE][ID]
+        source_data = get_entity_data(source_id)
+        source_archives_id = source_data[CLAIMS][ARCHIVES_ID_PROPERTY][0][MAINSNAK][DATAVALUE][VALUE]
+        archives_ids.append(source_archives_id)
+
+    return harvesting_date, archives_ids
+
+
+def harvest_latest_versions(archives_ids):
+    client = storage.Client()
+    latest_versions = {}
+    for archives_id in archives_ids:
+        bucket_id = client.lookup_bucket(
+            LATEST_BUCKET_PATH.format(source_archives_id=archives_id)
+        )
+        if bucket_id is not None:
+            bucket = client.get_bucket(
+                LATEST_BUCKET_PATH.format(source_archives_id=archives_id)
+            )
+            blobs = client.list_blobs(bucket.name)
+            for blob in blobs:
+                archives_url = LATEST_URL.format(
+                    source_archives_id=archives_id,
+                    blob_name=blob.name
+                )
+                latest_versions[archives_id] = archives_url
+    return latest_versions
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="Harvesting script for the latest versions. Python 3.9.")
+    parser.add_argument(
+        "-a",
+        "--archives-ids-file",
+        action="store",
+        help="Name of the archives ids file. If the file does not exist, the script will create it.",
+    )
+    parser.add_argument(
+        "-l",
+        "--latest-versions-file",
+        action="store",
+        help="Name of the latest versions file. If the file exists, it will be overwritten.",
+    )
+    parser.add_argument(
+        "-d",
+        "--data-path",
+        action="store",
+        default="./data/",
+        help="Data path.",
+    )
+    parser.add_argument(
```

Comment:
Should we remove this CLI parameter, since we will only harvest `GTFS` catalogs here.

Reply:
Won't we harvest GBFS datasets in the future? I can remove it if not

Reply:
Probably, but no in this repository.

Reply:
Removed!